<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PMH::Internal::PmhModeWidget</class>
 <widget class="QWidget" name="PMH::Internal::PmhModeWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>568</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="Utils::MiniSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="treeViewLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="Views::TreeView" name="treeView">
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QScrollArea" name="scrollArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents_3">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>270</width>
         <height>280</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <widget class="QStackedWidget" name="stackedWidget">
          <property name="currentIndex">
           <number>2</number>
          </property>
          <widget class="QWidget" name="pageSynthesis">
           <layout class="QGridLayout" name="gridLayout_3">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item row="0" column="0">
             <widget class="QTextBrowser" name="pmhSynthesisBrowser"/>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="pageEditor">
           <layout class="QVBoxLayout" name="verticalLayout">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="PMH::PmhViewer" name="pmhViewer" native="true"/>
            </item>
            <item>
             <widget class="QDialogButtonBox" name="buttonBox">
              <property name="standardButtons">
               <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="formPage">
           <layout class="QGridLayout" name="gridLayout_5">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item row="0" column="0">
             <widget class="Form::FormDataWidgetMapper" name="formDataMapper" native="true"/>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Views::TreeView</class>
   <extends>QTreeView</extends>
   <header location="global">listviewplugin/treeview.h</header>
  </customwidget>
  <customwidget>
   <class>Utils::MiniSplitter</class>
   <extends>QSplitter</extends>
   <header location="global">utils/widgets/minisplitter.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Form::FormDataWidgetMapper</class>
   <extends>QWidget</extends>
   <header location="global">formmanagerplugin/formdatawidgetmapper.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PMH::PmhViewer</class>
   <extends>QWidget</extends>
   <header location="global">pmhplugin/pmhviewer.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
